robottelo/cli/factory.py

```diff
@@ -3942,6 +3943,9 @@ def virt_who_hypervisor_config(
     :param bool exec_one_shot: whether to run the virt-who one-shot command
         after startup
     :param bool upload_manifest: whether to upload the organization manifest
+    http://sesame.lab.eng.rdu2.redhat.com/pub/rhel7.repo
+    :param list added_repos: (Optional) a list of repositories dict options to setup additionally.
```

Comment:
i'd vote for `extra_repos` then, or smth else if you have better ideas. 'added repos' sounds like the repos which are already added to me, which is misleading.